### ISSUE TITLE
Dash - Needs Update or Removal

### ISSUE THREAD

**Issue by astupidmoose** [btcpayserver/btcpayserver#2088]
Dash recently upgraded to a new version, which effectively orphaned anyone running BTCpay onto a separate chain. This means BTCPay users can no longer use Dash for accepting payments or as a wallet. I am surprised no one has brought this up... maybe I am the only one using it? 

I am going to tag @snogcel and @BenjaminNitschke as I believe both have been involved in Dash upgrades in the past, to see if anyone would be able to bump the docker image we have to the newest iteration of 0.16.1

If not, then I suggest we remove Dash from the list of supported Altcoins since it's effectively unusable at the moment and will only cause headaches if not fixed.

**Comment by astupidmoose** [comment]
Attempting to solve my own problems:
https://github.com/btcpayserver/dockerfile-deps/pull/14 & https://github.com/btcpayserver/btcpayserver-docker/pull/403

**Comment by Kukks** [comment]
both merged, let us know how it went

**Comment by astupidmoose** [comment]
OK so I ran into a few issues with updating but it seems to be OK now. I'm going to close this ticket and document the steps taken incase anyone stumbles across this. 

1. Updated BTCPay, Dash would not sync (I believe because I had already been on the orphaned chain)
2. Remove _data directory and created again (emptying contents of directory probably just as good)
3. Rebooted dash
4. I noticed that Dash had fully synced, but no outbound transactions that were broadcast were synced, only inbound. I created a new dash wallet and input the new xpub into BTCPay. 
5. Dash now seems to be syncing both Inbound and outbound on the new wallet and customers are now able to pay in dash again. 
6. ????
7. Profit!
